source/server-discovery-and-monitoring/tests/rs/null_election_id-pre-6.0.yml

```diff
@@ -15,7 +15,7 @@ phases: [
                 setVersion: 1,
                 setName: "rs",
                 minWireVersion: 0,
-                maxWireVersion: 7
+                maxWireVersion: 16
```

Comment:
Wire version 16 is server 5.3

Reply:
The Java and C drivers are able to pass these tests still, I think because it's only in server selection where the maxWireVersion is checked.  Do you expect other drivers to fail these tests, and if so, what causes the failure?

Reply:
My expectation is that no drivers would fail these tests as they are for pre 6.0 and this tops the wire version for 5.3

Reply:
OK, this seems like a harmless change to make, even for drivers using submodules.  There is no loss of test coverage, and the tests should pass all drivers.

But why stop at 5.3? Any reason not to just bump it up the MongoDB 8.0 wire version?

Reply:
Oh, never mind.  I see below that you did that already.

Reply:
Oh, as you explained in Slack, this is for pre-6.0